security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+First, every bit of information may bring the adversary closer to their goal.
+It may not be necessary to shrink the anonymity set to the size of one.
+For instance, if the adversary plans a targeted DoS attack and can take down 100 servers, the anonymity set of 100 suffices.
+Second, the adversary can cross-correlate information from different sources.
+Even if a privacy leak looks relatively benign, we never know what it can achieve in combination with other data sources.
+Finally, especially in cryptographic settings, the attacker always has the "last resort" of brute force search.
+Cryptographic primitives are designed so that it is practically impossible to guess a secret such as a private key.
+Nevertheless, each bit of information brings the adversary closer to this goal, and at some point, it becomes attainable.
+
+In terms of Lightning, deanonymizing generally means deriving a correspondence between payments and users identified by node IDs.
+Each payment may be assigned a sender anonymity set and a receiver anonymity set.
+Ideally, the anonymity set consists of all the users of the network.
+This assures that the attacker has no information whatsoever.
+However, the real network leaks information that allows an attacker to narrow down the search.
+The smaller the anonymity set, the higher the chance of successful deanonymization.
+
+
+=== Differences between LN and Bitcoin in terms of privacy
+
+Bitcoin was initially perceived as an anonymous currency.
+Indeed, Bitcoin addresses are not linked to real identities.
+However, all transactions are broadcast in cleartext and can be analyzed.
+Multiple companies deanonymize users of Bitcoin and other cryptocurrencies.
+
+At first glance, Lightning provides better privacy than Bitcoin because Lightning payments are not broadcast to the whole network.
+While this improves the privacy baseline, other properties of the Lightning protocol may make anonymous payments more challenging.
+For instance, larger payments have fewer routing options.
+This may allow an adversary who controls well-capitalized nodes to route most large payments and discover payment amounts and probably other details.
+Another relevant difference between Lightning and Bitcoin is that Lightning nodes maintain a permanent identity, whereas Bitcoin nodes do not.
+A Bitcoin user can easily switch nodes used to receive blockchain data and broadcast transactions.
+A Lightning user, on the contrary, sends and receives payments through the nodes they have opened channels with.
+Moreover, the Lightning protocol assumes that routing nodes announce their IP address in addition to their node ID.
+This creates a permanent link between node IDs and IP addresses, which may be dangerous, considering that an IP address is often an intermediary step in anonymity attacks linked to the user's physical location and, in most cases, real-world identity.
+It is possible to use Lightning over Tor, but many nodes do not use this functionality.
```

Comment:
I think this statement needs to be cited, from my PoV I see Tor usage rising by the week for new nodes.

Reply:
According to https://1ml.com/statistics: 

- Number of Nodes: 15,079
- Tor Onion Service Nodes: 2,715

Reply:
Added a reference to https://1ml.com/statistics